libs/ballot-interpreter/src/hmpb-rust/timing_marks.rs

```diff
@@ -1298,6 +1316,29 @@ pub fn find_complete_from_partial(
         );
     }
 
+    // We only check for stretching along the vertical axis because that's the
+    // direction we scan ballots in. If we ever begin scanning horizontally, we
+    // should also check for stretching along the horizontal axis.
+    let max_allowed_vertical_distance = median_vertical_distance * 1.5;
```

Comment:
It seems high risk to try to interpret when there's any stretching, and I think it happens pretty rarely, so I'd suggest shortening the max gap allowed

Reply:
Shrunk to 120%.